projects/graph/src/graph.py

```diff
@@ -8,8 +8,8 @@ def __init__(self):
         pass  # TODO
         self.vertices = {}
     
-    def add_vertex(self, vertex_id):
-        self.vertices[vertex_id] = Vertex(vertex_id)
+    def add_vertex(self, vertex_id, vertices):
```

Comment:
I'm a little unclear on this part. I don't think we need the second argument, vertices, since we can assume that the vertex being added is being added to the graph's own vertices dictionary attribute.